src/UpdatedPackagesCollector.js

```diff
@@ -151,6 +151,7 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
+    // TODO: Ugh... something about this.
```

Comment:
perhaps a `__GLOBAL__` or `all` or similar key could be used to apply to all commands?